### PR TITLE
fix(trading): Min selector confusing

### LINKED ISSUE #1012: [BUG]: Min volume selector is confusing
At the moment it's not clear what dots do and there is no visual representation of min trading volume (as well as an explanation of what min volume is/using for).

I think we can make it an optional input field (with activation way similar to custom dpow protection rules) with an indicator in percents from 0%  to 100% from order amount

### PR DESCRIPTION
**Pull request by SylEze**
Closes #1012

### CODE REVIEW

**Comment by smk762** [comment]
If sliding up the minimum, then unticking checkbox, minimum stays but moveable dot hidden.
image

Expected: If not using custom minimum, line and value should revert to 0.

**Comment by ssakone** [comment]
@smk762 can you test again after CI?

File: atomic_defi_design/qml/Exchange/Trade/TradeBox/OrderForm.qml [hunk 1]
```diff
@@ -230,6 +233,18 @@ FloatingBackground {
...
+                        _volumeRange.oldFirstValue =  _volumeRange.defaultFirstValue
+                    } 
+                    else { 
+                        _volumeRange.defaultFirstValue = API.app.trading_pg.orderbook.current_min_taker_vol
+                        _volumeRange.first.value = API.app.trading_pg.orderbook.current_min_taker_vol
+                    }
+                }
+                onCheckedChanged: {
+                    
+                }
```
Comment (Milerius) [inline atomic_defi_design/qml/Exchange/Trade/TradeBox/OrderForm.qml:247]: What the purpose of this code ? Why do you push code that literally does nothing ?

**Review by smk762 — DISMISSED** [review]
> If sliding up the minimum, then unticking checkbox, minimum stays but moveable dot hidden.
> image
> 
> Expected: If not using custom minimum, line and value should revert to 0.

Confirm working as expected now

File: atomic_defi_design/qml/Exchange/Trade/TradeBox/OrderForm.qml (hunk 1 above)
Comment (ssakone) [reply atomic_defi_design/qml/Exchange/Trade/TradeBox/OrderForm.qml]: sorry.

**Pull request merged**